packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -125,16 +137,16 @@ class _ScrollbarState extends RawScrollbarState<Scrollbar> {
 
     return MaterialStateProperty.resolveWith((Set<MaterialState> states) {
       if (states.contains(MaterialState.dragged))
-        return dragColor;
+        return _scrollbarTheme.thumbColor?.resolve(<MaterialState>{MaterialState.dragged}) ?? dragColor;
```

Comment:
Unless I'm missing something, this expression, and the others like it, should be
```dart
return _scrollbarTheme.thumbColor?.resolve(states) ?? dragColor;
```

Reply:
I was wondering! I noticed in another class the theme was used this way to kind of 'seed' states I guess, I'll update.